### PR TITLE
Create a documentation for ChIMES MD

### PR DESCRIPTION
**Pull request by Yanjun-Lyu**
## Description

Created the documentation for ChIMES MD

### CODE REVIEW

File: doc/source/quick_start_chimes_md.rst
Comment (RKLindsey) [inline doc/source/quick_start_chimes_md.rst:1]: running

File: doc/source/quick_start_chimes_md.rst [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+    cp /path/to/chimes_lsq/test_suite-lsq/test_suite-md/h2o-2bcheby/params.cheby.txt          /my/tutorial_folder
+    cp /path/to/chimes_lsq/test_suite-lsq/test_suite-md/h2o-2bcheby/run_md.in          /my/tutorial_folder
+    cp /path/to/chimes_lsq/test_suite-lsq/test_suite-md/h2o-2bcheby/input.xyz          /my/tutorial_folder
+    
+    
+Step 2: Inspect the input
+*********************************************    
+
+Aside from the ``chimes_md`` code, a minimum of three input files are needed to run the MD code: A trajectory file, parameters file and a ``chimes_md`` configuration file (``input.xyz``, ``params.cheby.txt`` and ``run_md.in`` in this example, respectively). By running ``head input.xyz`` in the tutorial folder, one can see that the trajectory file is in an extended xyz format, where for a given frame, the first line provides the number of atoms, the next line, the three box lengths (Å), and the remaining lines provide an atom type, *x*, *y*, and *z* coordinates (Å), and the corresponding *x*, *y*, and *z* components of the forces acting on those atoms, in Hartree (H) per Bohr (B). Frame information is repeated in this format :math:`n_{\mathrm{frames}}` times.
```
Comment (RKLindsey) [inline doc/source/quick_start_chimes_md.rst]: Need to specify possible .xyz formats somewhere (box length line)

File: doc/source/quick_start_chimes_md.rst [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    ###################################
+    #### GENERAL CONTROL VARIABLES ####
+    ###################################
+ 
+ 
+    # RNDSEED # ! Seed. If not specified, default value 123457 is used
+    12357
+    # TEMPERA # ! In K
+    2000.0
+    # CMPRFRC # ! Compare computed forces against a set of input forces? ...If true, provide name of the file containing the forces for comparison
```
Comment (RKLindsey) [inline doc/source/quick_start_chimes_md.rst]: Don't just use default example, use minimum working example -- get rid of this line/option

File: doc/source/quick_start_chimes_md.rst [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+    false
+    # TIMESTP # ! In fs
+       0.025
+    # N_MDSTP # ! Total number of MD steps
+     16000
+    # NLAYERS # ! x,y, and z supercells.. small unit cell should have >= 1
+      2
+    # USENEIG #
```
Comment (RKLindsey) [inline doc/source/quick_start_chimes_md.rst:66]: Don't just use default example, use minimum working example -- get rid of this line/option

File: doc/source/quick_start_chimes_md.rst [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+ 
+    ###################################
+    ####    SIMULATION  OPTIONS    ####
+    ###################################
+ 
+    # VELINIT # (options are READ or GEN)
+      READ
+    # CONSRNT # (options are HOOVER <hoover time> or VELSCALE <scale freq>)
+      NVT-MTK HOOVER 50
+    !# CONSRNT # (options are HOOVER <hoover time> or VELSCALE <scale freq>)
```
Comment (RKLindsey) [inline doc/source/quick_start_chimes_md.rst]: Not sure what's happening here but should only have one CONSRNT

File: doc/source/quick_start_chimes_md.rst [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+    !      NVE
+    # PRSCALC # (options are ANALYTICAL or NUMERICAL)
```
Comment (RKLindsey) [inline doc/source/quick_start_chimes_md.rst]: Don't just use default example, use minimum working example -- get rid of this line/option

File: doc/source/quick_start_chimes_md.rst [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+      ANALYTICAL
+    ###################################
+    ####      OUTPUT  CONTROL      ####
+    ################################### 
+     # WRPCRDS # 
+      false 
+    # FRQDFTB # ! Frequency to output the DFTB gen file can also be called FRQTRAJ
```
Comment (RKLindsey) [inline doc/source/quick_start_chimes_md.rst]: Call this frqtraj because more intuitive/general

File: doc/source/quick_start_chimes_md.rst [hunk 7]
```diff
@@ -0,0 +1,121 @@
...
+      20
+    # FRQENER # ! Frequency to output energies
+      10 
+    # PRNTFRC # ! Print computed forces? Forces are printed to force_out.txt
```
Comment (RKLindsey) [inline doc/source/quick_start_chimes_md.rst]: Edge case - remove option

Comment (RKLindsey) [reply doc/source/quick_start_chimes_md.rst]: In short, don't just have user grab from test suite (which has many strange options) -- make your own example files for them to grab and update this accordingly

File: doc/source/quick_start_chimes_md.rst [hunk 8]
```diff
@@ -0,0 +1,121 @@
...
+      10 
+    # PRNTFRC # ! Print computed forces? Forces are printed to force_out.txt 
+      false
+
+    # ENDFILE #
+      
+.. Note::
+
+    * The box dimensions are read from the comment line of the xyz.
+    * The box dimsnsion can be int two different format; Non-orthogonal which is denpted as ``NON_ORTHO <xx> <xy> <xz> <yx> <yy> <yz> <zx> <zy> <zz>``, orthogonal which is ``<x> <y> <z>``. The default is orthogonal.
```
Comment (RKLindsey) [inline doc/source/quick_start_chimes_md.rst]: Typo (dimension)

Non ortho is ax ay az bx by bz... an specify that those are cell parameters

Remove last sentence

File: doc/source/quick_start_chimes_md.rst [hunk 9]
```diff
@@ -0,0 +1,121 @@
...
+    
+    /path/to/chimes_lsq/build/chimes_md-serial run_md.in > run_md.log
+
+To run the MD simulation in parallel, run the following command:
+.. code-block::
+    
+    /path/to/chimes_lsq/build/chimes_md run_md.in > run_md.log
+
+This will generate run_md.out, traj_bad_r.lt.rin.xyz, traj_bad_r.lt.rin+dp.xyz, traj_bad_r.ge.rin+dp_dftbfrq.xyz, traj.xyz, run_md.out, restart.xyzv ,restart.bak ,output.xyz , and md_statistics.out, but the most important files are ``traj.xyz`` , ``run_md.out`` and ``md_statistics.out``.
+``traj.xyz`` is printed based on  ``FRQDFTB`` aka ``FRQTRAJ`` which is the trajectory output frequency, ``run_md.out`` is the main output of MD simulation, and ``md_statistics.out`` is the statistics of the MD simulation printed every ``FRQENER``.
```
Comment (RKLindsey) [inline doc/source/quick_start_chimes_md.rst]: Get rid of any mention of FRQDFTB

Also change run_md.log to run_md.log


This will generate run_md.out, traj_bad_r.lt.rin.xyz, traj_bad_r.lt.rin+dp.xyz, traj_bad_r.ge.rin+dp_dftbfrq.xyz, traj.xyz, run_md.out, restart.xyzv ,restart.bak ,output.xyz , and md_statistics.out. Of these files, the most important are ``traj.xyz`` , ``run_md.out`` and ``md_statistics.out``, which provide the simulation trajectory, running information on the simulation, and a running log of simulation properties, respectively.  The ``traj.xyz`` and ``md_statistics.out`` are output every ``FRQTRAJ`` and ``FRQENER`` simulation steps, respectively.

**Review by RKLindsey — CHANGES_REQUESTED** [review]
See comments - @S-A-Almohri  can help with this (was here for comments)

**Also, change your PR to target the LL develop branch and not main**

**Comment by S-A-Almohri** [comment]
@RKLindsey Addressed the requested changes in commit d1ff3fd on @Yanjun-Lyu's fork; @Yanjun-Lyu  please check the RST formatting one more time before we merge

**Review by RKLindsey — APPROVED** [review]
